### PR TITLE
Document kubeadm --skip-preflight-checks.

### PR DESCRIPTION
**Pull request by dgoodwin**
cc @kubernetes/sig-cluster-lifecycle

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1521)

### CODE REVIEW

File: docs/admin/kubeadm.md [hunk 1]
```diff
@@ -112,6 +117,12 @@ to change the DNS name suffix. Again, you will need to update the
 `/etc/systemd/system/kubelet.service.d/10-kubeadm.conf` file accordingly else DNS will
 not function correctly.
 
+- `--skip-preflight-checks`
+
+By default, 'kubeadm' runs a series of preflight checks to validate the system
```
Comment (dgoodwin) [inline docs/admin/kubeadm.md]: My mistake, updated now.

**Comment by luxas** [comment]
Needs a rebase yet @dgoodwin

**Comment by dgoodwin** [comment]
Rebased.

**Comment by luxas** [comment]
LGTM

**Pull request merged**